tenseal/proto/tensealcontext.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package tenseal;
+
+
+//TenSEAL Private parameters
+message TenSEALPrivateProto {
+    // Serialized secret key
+    bytes secret_key = 1;
+    // Relin keys flag
+    bool relin_keys = 2;
+    // Galois keys flag
+    bool galois_keys = 3;
```

Comment:
the name is somehow misleading here, it seems like we are talking about the keys themselves, where it's whether the keys are generated or not, can we rename to key_generated or something similar?